trajopt_ifopt/src/continuous_collision_evaluators.cpp

```diff
@@ -0,0 +1,360 @@
+/**
+ * @file continuous_collision_evaluators.h
```

Comment:
Not reviewing this since I assume it was just copied. Let me know if that isn't true

Reply:
The only thing that has changed other than what is mentioned above, is the hashing of the data.